src/components/schedule-visualizer/tests/robot-default-icon.test.tsx

```diff
@@ -0,0 +1,35 @@
+import { mount } from 'enzyme';
+import React from 'react';
+import { makeRobot } from '../../../mock/utils';
+import ColorManager from '../colors';
+import RobotDefaultIcon from '../robot-default-icon';
+
+describe('RobotDefaultIcon', () => {
+  const colorManager = new ColorManager();
+  // TextEncoder is not available in node
+  colorManager.robotColor = jest.fn(async () => 'black');
+  colorManager.robotColorFromCache = jest.fn(() => 'black');
+
+  test('renders correctly', async () => {
+    const root = mount(
+      <svg>
+        <RobotDefaultIcon robot={makeRobot()} footprint={1} colorManager={colorManager} />
+      </svg>,
+    );
+    expect(root).toMatchSnapshot();
```

Comment:
-> `expect(toJson(root)).toMatchSnapshot();` you need to convert the container to Json with `enzyme-to-json` library first, an then compare with the snapshot. If you don't make that conversion the snapshot will be an empty object. You can check that here `src/components/schedule-visualizer/tests/__snapshots__/robot-default-icon.test.tsx.snap`

exports[`RobotDefaultIcon renders correctly 1`] = `ReactWrapper {}`;